doc/advanced.rst

```diff
@@ -147,6 +154,9 @@ package in a module called ``scraper``. Here is the scraper code::
        def __init__(self):
            self.seen = set()
 
+       def __repr__(self):
+           return 'PNGScraper'
+
        def __call__(self, block, block_vars, gallery_conf):
```

Comment:
Just a quick note that I ran into when figuring out gif stuff with @stefanv - we should add somewhere an explanation of the inputs ("block, block_vars, gallery_conf"). We couldn't easily find this information and it was causing confusion for the scikit-image folks

Reply:
Agreed, let's do it separately though. Feel free to add an issue.